test/unit-global/frozenPromise.js

```diff
@@ -0,0 +1,42 @@
+
+'use strict'
+
+/* eslint-disable no-native-reassign */
+
+// this is what we're testing
+require('../../app/scripts/lib/freezePromise')
+
+const test = require('tape')
```

Comment:
Hah, interesting. It looks like this is the only place we use `tape`? It was already in the manifest, but we use mocha everywhere else.

Reply:
Hah, yeah I needed something lightweight and didn't want to install `tape` so you can bet I was excited to see it was already in the manifest 😄

Reply:
Is mocha less "lightweight"? I believe mocha would work just as well here, replacing `test` with `it`.

I have no objection to using `tape` here, but I'll probably end up proposing we remove it in a future PR to reduce our dependency count. I've been meaning to go through our dependencies to remove the ones we don't need.

Reply:
mocha is less lightweight in that it controls the environment (thats where `describe` etc come from) and the process is run in mocha. tape is lightweight in the sense you just run it in node or browser or wherever

Reply:
Originally I just used `tape` heuristically because the normal unit test environment didn't work. However, the problem was not `mocha` but our environment setup, specifically when we reassign the global Promise. I removed `tape` and started using plain `mocha` for the tests I added.